third_party/babel/custom-babel-helpers.js

```diff
@@ -0,0 +1,88 @@
+(function (global) {
```

Comment:
@cramforce thanks. will merge this AFTER our current next release so it can sit for a while.